.idea/dictionaries/gustorn.xml

```diff
@@ -0,0 +1,7 @@
+<component name="ProjectDictionaryState">
```

Comment:
What is this file for?

Reply:
How can it help? It's some kind of dictionary that is not necessary for idea. Also it seems it's gustorn's personal file since it has same name. I'd rather remove it from PR before merge.

Reply:
Yup, I think it can be wiped out, especially since the dictionary contains 1 item only.
Done.